src/faldbt/project.py

```diff
@@ -51,13 +51,16 @@ def __post_init__(self):
         node = self.node
         self.unique_id = node.unique_id
         if node.resource_type == NodeType.Test:
-            self.name = node.test_metadata.name
-
-            # node.test_metadata.kwargs looks like this:
-            # kwargs={'column_name': 'y', 'model': "{{ get_where_subquery(ref('boston')) }}"}
-            # and we want to get 'boston' is the model name that we want extract
-            self.model = re.findall(r"'([^']+)'", node.test_metadata.kwargs['model'])[0]
-            self.column = node.test_metadata.kwargs['column_name']
+            if hasattr(node, "test_metadata"):
+                self.name = node.test_metadata.name
+
+                # node.test_metadata.kwargs looks like this:
+                # kwargs={'column_name': 'y', 'model': "{{ get_where_subquery(ref('boston')) }}"}
+                # and we want to get 'boston' is the model name that we want extract
+                self.model = re.findall(r"'([^']+)'", node.test_metadata.kwargs['model'])[0]
+                self.column = node.test_metadata.kwargs['column_name']
+            else:
+                logger.warn(f"Parsing of this test not supported: {node.name}")
```

Comment:
`Parsing of this test` what does this mean exactly?

Reply:
The message should be something like "Only generic tests are processed by fal. This test appears to be not generic: TEST_NAME"

Reply:
Good point!